pkg/mock/v1alpha3/api/manager.go

```diff
@@ -7,88 +7,88 @@ package mock
 import (
 	context "context"
 	gomock "github.com/golang/mock/gomock"
-	v1alpha3 "github.com/kubeflow/katib/pkg/apis/manager/v1alpha3"
+	api_v1_alpha3 "github.com/kubeflow/katib/pkg/apis/manager/v1alpha3"
```

Comment:
Do you know, which version of mockgen are you using?
Why it is updated few naming ?

Reply:
I believe I use the master branch of mockgen.